systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -363,6 +373,50 @@ public void testForUpdateValuesInConfigMap() {
         }
     }
 
+    @Test
+    @Resources(value = "../examples/templates/cluster-controller", asAdmin = true)
+    @OpenShiftOnly
```

Comment:
What is the motivation for having this test as OpenShift only just because it uses template instead of using config map directly? It means that this test will not run on Travis without having any actual benefit from it.